games/gopsHelpers.js

```diff
@@ -72,17 +78,78 @@ function roundWinner(instance) {
   }
 };
 
+//Awards points to the winner of the round. Takes roundWinner as an argument.
 function awardPoints(instance, id) {
 
   if (id) {
     instance[id].points += instance.prizePool[0];
     instance.prizePool.splice(0, 1);
+    reset(instance);
   } else {
     let message = "Round nulled, two or more players played same card."
     instance.prizePool.splice(0, 1);
+    reset(instance);
+    return message;
   }
 }
 
+//Once winner has been decided, resets hand of each player to zero.
+function reset(instance) {
+
+  for (let player in instance) {
+    if(instance[player].readyCard !== undefined){
+      instance[player].readyCard = 0
+    }
+  }
+};
+
+//At the begginning of every round, will check to see if every player has played. Status conditional -- needs to run
+//before checking a winner.
+function readyCheck(instance) {
+
+  let ready = true;
+  for (let player in instance) {
+    if(instance[player].readyCard === 0) {
+      ready = false;
```

Comment:
If we receive 1 false, this will fail. Instead, we could `return false` here. On line 116, `return true`. This way we don't have to continue looping through players.